test/code/scxsystemlib/memory/memoryinstance_test.cpp

```diff
@@ -695,9 +710,9 @@ class MemoryInstance_Test : public CPPUNIT_NS::TestFixture
                 }
 
                 keyValues["TotalMemory"]     = ToSCXUlong(topTokens[2 + toffset]);
+                keyValues["TotalSwap"]       = ToSCXUlong(topTokens[12 + toffset]);
```

Comment:
Please change ordering to eliminate needless code churn. This needs to be town lines down, it looks like.